geth/jail/ottoext/fetch/fetch_test.go

```diff
@@ -0,0 +1,226 @@
+package fetch
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/robertkrimen/otto"
+
+	"github.com/status-im/status-go/geth/jail/ottoext/loop"
+)
+
+func must(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
```

Comment:
Please, use testify require everywhere and don't invent new assertion means nor use panic.

Reply:
Ouch, those tests need to be completely revisited and rewritten.